config.yml

```diff
@@ -0,0 +1,247 @@
+defaults:
```

Comment:
Every time I see the old `js` config turned into the new `yml` config I get a happy tear in my eye. 💯